scripts/pkg-tools/show-unpkg-files

```diff
@@ -86,20 +85,20 @@ create_complete_filelist()
 
 filter_out_pkg_files_from_cache()
 {
-    zcat $ROOT_CACHE                             \
-        | cache-exclude $FILELIST                \
-        | egrep -v '^.\s+/(home|tmp|root|var)/'  \
-        | egrep -v '^.\s+/usr/lib/sysimage/rpm/' \
-        | cache-kill-empty-dirs                  \
-        | gzip                                   \
-        >$UNPKG_CACHE
+    zcat "$ROOT_CACHE"                             \
+        | cache-exclude "$FILELIST"                \
+        | grep -E -v '^.\s+/(home|tmp|root|var)/'  \
```

Comment:
Why prefer `grep -E` over `egrep`?

`egrep` has been around for at least 3 (probably more) decades. While on the man page it says it's deprecated, this is pure self-importance on the part of whoever of the FSF wrote that, and no more than an idle threat: The moment they actually remove it, not only will hell freeze over, a megaton of existing shell code will break; and the backlash will be so hard that they will have to publicly apologize and make it available again within mere hours of the removal.

Reply:
`egrep` emits a message to stderr https://lists.gnu.org/archive/html/info-gnu/2022-09/msg00001.html